pyproject.toml

```diff
@@ -14,7 +14,7 @@ target-version = ["py36"]
 [tool.mypy]
 show_error_codes = true
 check_untyped_defs = true
-no_implicit_reexport = true
+implicit_reexport = false
```

Comment:
https://mypy.readthedocs.io/en/stable/config_file.html#inverting-option-values

`no_implicit_reexport` is also a valid flag actually, but yours works as well.

Reply:
Ah, so it is. Just found it weird that the documentation has an `implicit_reexport` and a `no_implicit_optional`.

Reply:
If you wouldn't mind reverting this I'm happy to merge the other change!